sql/src/test/java/io/crate/integrationtests/PartitionedTableIntegrationTest.java

```diff
@@ -2201,41 +2188,10 @@ public void testMultipleWritesWhenOnePartitionIsReadOnly() {
     @Test
     public void testCurrentVersionsAreSetOnPartitionCreation() throws Exception {
         execute("create table doc.p1 (id int, p int) partitioned by (p)");
-        writeOldVersionsToTemplate();
 
         execute("insert into doc.p1 (id, p) values (1, 2)");
-        execute("select version['created']['cratedb'] from information_schema.table_partitions where table_name='p1'");
-
-        assertThat(response.rows()[0][0], is(Version.CURRENT.number()));
-    }
-
-    private void writeOldVersionsToTemplate() throws Exception {
-        String templateName = PartitionName.templateName("doc", "p1");
-        GetIndexTemplatesResponse templatesResponse = admin().indices()
-            .getTemplates(new GetIndexTemplatesRequest(templateName))
-            .actionGet(5, TimeUnit.SECONDS);
-        IndexTemplateMetaData templateMetaData = templatesResponse.getIndexTemplates().get(0);
-        Map<String, Object> mapping = AlterTableOperation.parseMapping(
-            templateMetaData.mappings().get(DEFAULT_MAPPING_TYPE).toString());
-
-        Map<String, Object> metaMap = IndexMappings.getMetaMapFromMapping(mapping);
-        assert metaMap != null : "could not resolve _meta map from template";
-        Map<String, Object> versionMap =  MapBuilder.<String, Object>newMapBuilder()
-            .put(CRATEDB_VERSION_KEY, 2031299)
-            .put(ES_VERSION_KEY, 5060899)
-            .map();
-        metaMap.put(VERSION_STRING, versionMap);
-
-        PutIndexTemplateRequest templateRequest = new PutIndexTemplateRequest(templateName)
-            .patterns(templateMetaData.patterns())
-            .settings(templateMetaData.settings())
-            .order(templateMetaData.order());
-
-        for (ObjectObjectCursor<String, AliasMetaData> container : templateMetaData.aliases()) {
-            templateRequest.alias(new Alias(container.key));
-        }
-        templateRequest.mapping(DEFAULT_MAPPING_TYPE, mapping);
+        execute("select version['created'] from information_schema.table_partitions where table_name='p1'");
 
-        admin().indices().putTemplate(templateRequest).actionGet(5, TimeUnit.SECONDS);
+        assertThat(response.rows()[0][0], is(Version.CURRENT.externalNumber()));
```

Comment:
Does this test still makes sense with these modifications? If it is ensured that the indexsettings are not taken from the template or are not written to the template, we could remove it. Otherwise we'd have to change it.

Reply:
Not sure, maybe we should create a test case in the `crate-qa` bwc tests for this?